### ISSUE TITLE
Minimize bar obsures video chat boxes

### ISSUE THREAD

**Issue by benbarbour** [saif-ellafi/foundryvtt-minimal-ui#21]
The minimize bar overlaps the bottom of the video chat boxes. It would be great to avoid this.

![minimize bar with video](https://user-images.githubusercontent.com/487715/115309374-0c763c00-a121-11eb-8036-f23d3be5911b.png)

**Comment by saif-ellafi** [comment]
Thank you for sharing this feedback. Remember you have the option to move the minimized bar to the top too.
I'll think about what can be done in this regard.

**Comment by benbarbour** [comment]
Yeah, maybe I'll do that for now. Thanks for such a great module, btw!!

**Comment by saif-ellafi** [comment]
I am not sure how to solve this. Pushing the bar up when cameras are active is not a viable option.

Unfortunately I'll have to disable the bottom bar option when cameras are active. But it is a bit low priority since this can be already achieved through configuration.

Will keep this issue a bit longer open in case there are other ideas or solutions.

**Comment by benbarbour** [comment]
Can the video dock be bumped up?

**Comment by saif-ellafi** [comment]
> Can the video dock be bumped up?

That's actually not a bad idea. The trick would be to fit it above the Macro Hotbar but below the Video line... hmm might go to bed with this thought, thanks!

How do you like the top bar so far? it benefits from the most space so far.

**Comment by benbarbour** [comment]
Haven't got to play yet, but it looks nice in my testing! :)

**Comment by saif-ellafi** [comment]
Is this a valid issue, @benbarbour? Please re-open if so, it has been a long time now, and there is V9

**Comment by benbarbour** [comment]
Now the video windows obscure the minimized windows. I've since moved to top bar (and might be going with the new experimental permanent bar along the top), so it doesn't affect me, but it's still not nice behaviour. 🤷 

![image](https://user-images.githubusercontent.com/487715/151222449-01739bd9-f805-47a6-bd54-f2c85c1bdc61.png)

**Comment by saif-ellafi** [comment]
Thank you @benbarbour - but I wonder if this issue is with Minimal UI or Window Controls. Need to test a bit the camera features, I haven't been using them for a while.

**Comment by benbarbour** [comment]
Oh right. I think I filed this before they split.

**Comment by saif-ellafi** [comment]
oh! A historical bug then :D
